abtesting/ABTestingExample/Shared/AppConfig.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright 2021 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Firebase
+
+class AppConfig: ObservableObject {
+  @Published var colorScheme : ColorScheme
+  init() {
+    let value = RemoteConfig.remoteConfig()["color_scheme"].stringValue ?? "nil"
+    colorScheme = value == "dark" ? .dark : .light
```

Comment:
cleanup: rather than ternary, add an extension for `ColorScheme` with an initializer that takes in a String and sets dark and light accordingly

Reply:
Just did a basic implementation, please feel free to provide further suggestions on this next iteration